src/pages/introduction/scholarly.mdx

```diff
@@ -4,4 +4,90 @@ title: "SCHOLARLY INTRODUCTION: Dickens’s Working Notes"
+
+Serial form not only structured Dickens’s compositional habits as a novelist, but shaped the contours of his other professional commitments and the rhythms of his personal life. By the time Dickens came to write _Hard Times_ for publication in _Household Words_ in 1854, he found himself needing to conceive and compose the novel–as the Working Notes vividly illustrate–in the standard monthly installment size he had become accustomed to, before dividing those into smaller segments for weekly publication.
+
+The Working Notes are evidence and a robust material record of Dickens’s navigation of serial production. The complete sets of Notes–both for individual novels and across multiple novels from _Dombey and Son_ (1846-48) onwards–enable us to look back, retrospectively, and trace developments in Dickens’s craft. But the Notes, we believe, can also activate the contingency of serial composition and reanimate our engagement with the processual nature of Dickensian serial form. 
+
+To take a basic but illustrative example: the left-hand memoranda of many Working Notes show Dickens posing questions to himself, frequently about the potential inclusion of specific characters in a given number, followed by answers that are (very frequently) in a clearly different ink or nib. These inscriptions on the Notes–query, followed by answer–are traces of the temporal distension of Dickens’s compositional process. Read alongside the often heavily edited manuscript pages of the novel, they cultivate a sense of potentiality and counterfactual possibilities within each number, as Dickens moves month by month from initial conception to completed novel. The Working Notes, in other words, offer us a point of engagement with Dickens’s serial novels that unsettles the finality of the published number or the completed novel.  
+
+## Pattern and Process: Dickens as Story-Weaver
+
+The tension between finished form and compositional process was central to Dickens’s own experience of his serial novels. In his prefaces, which he wrote immediately after finishing a novel, he often used the metaphor of weaving to name the conflicting pressures of the “general purpose and design” and “the temptation of the current monthly number” (_Martin Chuzzlewit_ 5). In the preface to _Little Dorrit_ (1855-57), for instance, he made this plea to his readers:
+
+_[A]s it is not unreasonable to suppose that I may have held [the work’s] threads with a more continuous attention than anyone else can have given them during its desultory publication, it is not unreasonable to ask that the weaving may be looked at in its completed state, and with the pattern finished._ (xxi)
```

Comment:
Markdown has markup for block quotes that you might want to try. Put a ">" at the beginning of the first line of the quote to check it out.

Reply:
This is great--thank you @csbailey5t. These were two questions I had as I was doing it, so I will make these changes and merge. Anna and I can discuss the block quotations as a matter of style